Parsers/ASimWebSession/Parsers/ASimWebSessionCiscoFirepower.yaml

```diff
@@ -0,0 +1,204 @@
+Parser:
+  Title: Web Session ASIM parser for Cisco Firepower
+  Version: '0.1.0'
+  LastUpdated: Oct 30 2023
+Product:
+  Name: Cisco Firepower
+Normalization:
+  Schema: WebSession
+  Version: '0.2.6'
+References:
+- Title: ASIM Web Session Schema
+  Link: https://aka.ms/ASimWebSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Cisco Firepower Documentation
+  Link: 
+    https://github.com/CiscoSecurity/fp-05-firepower-cli/tree/master/estreamer
+    https://www.cisco.com/c/en/us/td/docs/security/firepower/620/api/eStreamer/EventStreamerIntegrationGuide/IS-DCRecords.html#88027
+    https://www.cisco.com/c/en/us/td/docs/security/firepower/620/api/eStreamer/EventStreamerIntegrationGuide/IS-DCRecords.html#84248
+Description: |
+  This ASIM parser supports normalizing commonly used Cisco Firepower messages collected using the CEF Data Connector to the ASIM WebSession normalized schema.
+ParserName: ASimWebSessionCiscoFirepower
+EquivalentBuiltInParser: _ASim_WebSession_CiscoFirepower
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventFieldsLookup = datatable(
+        DeviceAction: string, 
+        DvcAction: string,
+        EventResult: string
+    )
+        [
+        "Detect", "Allow", "Partial",
+        "Block", "Deny", "Failure",
+        "Malware Cloud Lookup", "Deny", "Failure",
+        "Malware Block", "Deny", "Failure",
+        "Malware Allow List", "Allow", "Success",
+        "Cloud Lookup Timeout", "Deny", "Failure",
+        "Custom Detection", "Allow", "Partial",
+        "Custom Detection Block", "Deny", "Failure",
+        "Archive Block-Depth Exceeded", "Deny", "Failure",
+        "Archive Block-Encrypted", "Encrypt", "Failure",
+        "Archive Block-Failed to Inspect", "Deny", "Failure"
+    ];
+    let DirectionLookup = datatable (CommunicationDirection: string, NetworkDirection: string)[
+        "1", "Inbound",
+        "2", "Outbound"
+    ];
+    let parser=(disabled: bool=false) {
+        CommonSecurityLog
+        | where not(disabled) 
+        | where DeviceVendor == "Cisco" and DeviceProduct == "Firepower"
+            and DeviceEventClassID in("File:500:1", "FileMalware:502:1", "FireAMP:125:1")
```

Comment:
@jayeshprajapaticrest @niralishah-crest 
Only these types of values have WebSession related logs? Also, can you please help sharing the logic behind categorizing it as an WebSession log.
DeviceEventClassID in("File:500:1", "FileMalware:502:1", "FireAMP:125:1")

Reply:
By looking at event descriptions and its field we have categorized it as websession events. As similar kind of event(accessing malicious file on web) we have considered in Cisco Meraki for WebSession.
![image](https://github.com/Azure/Azure-Sentinel/assets/107753570/7e52e2f7-d867-4d5f-8965-3dad1a28e91a)